exercises/tutorial_halfday/ex9_matrix-transpose-local-array.cpp

```diff
@@ -179,7 +179,7 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
   //
   RAJA::RangeSegment row_Range(0, N_r);
   RAJA::RangeSegment col_Range(0, N_c);
-
+#if 0
```

Comment:
Are these `if 0` entries intentional?

Reply:
nope, it's just work in progress..  I'm going to work towards removing those.